### ISSUE TITLE
Feature Request: Could we enter a kibitz from scratch?

### ISSUE THREAD

**Issue by GreenAsJade** [neagle/gokibitz#161]
This would be really useful for setting up a situation to discuss - joseki being the obvious example, but there are others.

For example, I was reading about [3-space line-3 invasion at Senseis](http://senseis.xmp.net/?InvasionOfThirdLineThreeSpaceExtension), where they have a [static picture with numbers counters to show the moves](http://senseis.xmp.net/?InvasionOfThirdLineThreeSpaceExtension#toc2).  Quite hard to follow.  How much better to have a link to gokibitz where a reader can step through the sequence under discussion, and contribute to a live discussion about it!!

**Comment by GreenAsJade** [comment]
... and, for bonus marks, how much even better if that could be embedded in the page, instead of a link away from the page to gokibitz!

**Comment by neagle** [comment]
I definitely see the use for something like this, but it's getting into the territory of adding full-on SGF editing capabilities to GoKibitz. It's something I'd love to see, but really involves a ton of new functionality. And it's not functionality that doesn't exist elsewhere... SGF editors of all stripes are great for this kind of thing. I've tried to avoid time-consuming reproduction of things that are available elsewhere.

That said, it's not impossible that it could come back somewhere down the road. Thanks for the suggestion!
